buddyboss/buddyboss-platform
# #Issue 1900 Loading issue fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #ISSUE_NUMBER .

### How to test the changes in this Pull Request:

1) Install WordPress 5.6
2) Go to a group with more than 10 members
3) Navigate to the Send Messages tab
4) Notice the error
### Proof Screenshots or Video



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
